### ISSUE TITLE
Load external CSS file in components.styles

### ISSUE THREAD

**Issue by IonicaBizau** [jillix/flow-nodejs#158]
``` json
{
   "components": {
      "styles": [
          "http://..."
      ]
   }
}
```

**Comment by ottiker** [comment]
An `external` path, for now, must look like `//domain.com/file.css` instead of `[protocoll]://domain.com.file.css`. But it should work in both ways.

**Comment by ottiker** [comment]
Fixed in af6736ce0415370d0ef5c4f6d010d1ef50dc8822.

**Comment by IonicaBizau** [comment]
This is still broken. :disappointed:

**Comment by ottiker** [comment]
@IonicaBizau can you post the `client.styles` composition config and the `url` that is generated in the browser?

**Comment by IonicaBizau** [comment]
I changed the configuration using a local file, but it was looking like below:

``` json
{
   "components": {
      "styles": [
          "https://foo..."
      ]
   }
}
```

And the link was:

```
http://github.jillix....:10001/https://foo
```

Something like that...

**Comment by ottiker** [comment]
Strange.. for me it's working.

**Comment by IonicaBizau** [comment]
@adioo Sorry, actually I was trying to load external JavaScript files. Is that supposed to work?

**Comment by ottiker** [comment]
The `js` files were not working. I fixed it in cf9c780656e8dad5145201be7c34d7d78a6c2d60.
@IonicaBizau Can you test it and close this issue?

**Comment by ottiker** [comment]
It works :+1:
